exes/mqtt-broadcaster/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+ add_executable(mqtt-broadcaster "src/main.cpp")
+ add_executable(mqtt-broadcaster-unit-tests "tests/unit_tests.cpp")
```

Comment:
most common pattern in repo is to have CMakeLists file under tests directory, it will be easier to remove targets with BUILD_TESTING cmake flag

Reply:
example 
```
if (BUILD_TESTING)
  add_subdirectory(tests)
endif ()
```

Reply:
Added a Cmake Lists file under the tests folder and the BUILD_TESTING option as well.